Modules/booking/Controller/BookingController.php

```diff
@@ -244,6 +244,7 @@ public function dayAction($id_space, $action, $message) {
         $calEntries = $modelEntries->getEntriesForPeriodeAndResource($id_space, $dateBegin, $dateEnd, $curentResource);
 
         // curentdate unix
+        $curentDate = (!$curentDate || $curentDate == "") ? $curentDate = date("Y-m-d") : $curentDate;
```

Comment:
? $curentDate = date(..)   ??
don't you mean

    $curentDate = (!$curentDate || $curentDate == "") ? date("Y-m-d") : $curentDate;

same for following ones

Reply:
ah yes, that's what I mean. I push back with correction